packages/flutter_test/lib/src/binding.dart

```diff
@@ -532,6 +531,7 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     }
 
     assert(inTest);
+    asyncBarrier(); // When using AutomatedTestWidgetsFlutterBinding, this flushes the microtasks.
```

Comment:
looks like we actually already have one here, so it's not needed after all (see line 519)